src/blocks.rs

```diff
@@ -0,0 +1,188 @@
+use crate::mesher::CubeFace;
+
+pub struct Block<'a> {
+    pub name: &'a str,
+    pub texture_names: [&'a str; 6],
+    pub is_solid: bool,
+}
+
+pub const BLOCKS: [Block; 14] = [
+    Block {
+        name: "air",
+        texture_names: ["air", "air", "air", "air", "air", "air"],
```

Comment:
Something should be done about the duplication of these texture names. Perhaps introduce all_faces(texture_name: &str) and top_side_bottom(texture_name: &str)?

Reply:
If all are the same just use `["air"; 6]`.